db/migrate/20161212003047_add_payments.rb

```diff
@@ -0,0 +1,10 @@
+# coding: utf-8
+# frozen_string_literal: true
+class AddPayments < ActiveRecord::Migration
+  def change
+    create_table :payments do |t|
+      t.integer :amount, null: false
```

Comment:
using integers here (halala) specifically avoids things like https://bugs.ruby-lang.org/issues/12958 and all machine epsilon errors in general